docs/versioned/admin/editing-configmaps.md

```diff
@@ -0,0 +1,155 @@
+---
+audience: administrator
+components:
+  - serving
+  - eventing
+function: explanation
+---
+# Editing ConfigMaps
+
+This page provides information and best practices for editing ConfigMaps. While there is no technical difference in how Knative uses ConfigMaps in Kubernetes, you should be aware of the following points and tips while working with ConfigMaps:
+
+- The _example key. Don't modify.
+- Value propagation. Know when changes to default settings take affect.
+- Version control. Use ConfigMaps for version control.
+
+The most common way to use ConfigMaps is to configure settings for containers running in a Pod, or in multiple pods, that use a specified namespace. There are a number of other ways to use ConfigMaps as described in [Kubernetes ConfigMaps](https://kubernetes.io/docs/concepts/configuration/configmap/) documentation.
```

Comment:
This seems misleading, as this is not how the Knative controllers use ConfigMaps.  Developers can still use ConfigMaps for these purposes, but that is a separate topic in a different area of the documentation (if needed).